src/features/tags/api/useCreateTagValueDefinition.ts

```diff
@@ -32,7 +32,7 @@ function buildKey({
 
 const createTagValueDefinition = post<
   { data: typeof TagValueDefinitionSchema.Encoded },
-  { label: string },
+  { label: string, displayName: string },
```

Comment:
I believe this can be optional
```suggestion
  { label: string, displayName?: string },
```

Reply:
Agree! Changed.